helm/security-pack/templates/apps.yaml

```diff
@@ -25,6 +25,7 @@ spec:
   name: {{ .chartName }}
   namespace: {{ .namespace }}
   version: {{ .version }}
+  {{- if $.Values.userConfig }}
```

Comment:
We need to check if userConfig is provided. Otherwise the `with` fails

```
reason: 'template: security-pack/templates/apps.yaml:44:15: executing "security-pack/templates/apps.yaml"
    at <$.Values.userConfig>: wrong type for value; expected map[string]interface
    {}; got interface {}'
```